### PR TITLE
Template fix

### PR DESCRIPTION
**Pull request by stephenpaul2727**
Embedded the information from https://help.rs.gsu.edu/display/PD/ACoRE on the partials template.
Linked to live page: https://hpcgateway.gsu.edu/

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -147,6 +147,11 @@ body{
 
 }
 
+.vertical-space {
+    padding-top:2%;
+    padding-bottom: 2%";
```
Comment (machristie) [inline assets/css/style.css]: You've got an extra double quote there.

File: partials/template.blade.php [hunk 2]
```diff
@@ -1,53 +1,127 @@
...
-		<div class="col-md-offset-2 col-md-8">
-			<h1 >Why Supercomputing?</h1>
-			<p style="color:#cccccc;">
-				Supercomputing simplifies and boosts modeling simulation, helps to make better use of computational expertise and create manufacturing apps.<br/>
-			</p>
-			<hr/>
+		<div class="col-md-6">
+			<p><strong>Blue Cluster</strong> (Originally known as Orion)</p>
+			<!-- Please use img-fluid instead of img-responsive if your bootstrap version if >=4.0 -->
+			<img class="center-image img-responsive" src="/assets/img/orion.png">
```
Comment (machristie) [inline partials/template.blade.php]: I think they want to link to the live image, not a static one. Same deal on line 83.

**Review by machristie — CHANGES_REQUESTED** [review]
Thanks Stephen. In general looks good, I just have a couple suggestions.

**Comment by stephenpaul2727** [comment]
Hi Marcus, I have made the requested changes. I had left the other two images as static because, I cannot find a live link for them when i inspect the page.

Here is a inspection of outer html:

``

As you can see its pointing to confluence download folder. so I went with downloading the image and serving it as static. Is this okay? Thanks

**Comment by machristie** [comment]
Hi @stephenpaul2727 , got it. I was looking at a page with Eroma yesterday and she had live links on the page.

@eroma2014 can you get the URLs to Stephen? Does he have a different link than we were looking at yesterday?

**Review by machristie — APPROVED** [review]

**Pull request merged**